src/models/account.cpp

```diff
@@ -507,6 +526,185 @@ int Account::importFromCSV(const std::string& path)
     return imported;
 }
 
+int Account::importFromOFX(const std::string& path)
+{
+    int imported{ 0 };
+    unsigned int nextId{ getNextAvailableTransactionId() };
+    const std::regex xmlTagRegex("^<([^/>]+|/STMTTRN)>(?:[+-])?([^<]+)");
+    std::smatch xmlTagMatch;
+    Transaction *transaction{ nullptr };
+    bool skipTransaction{ false };
+
+    std::ifstream file{ path };
+    if (file.is_open())
+    {
+        std::string line;
+        while (getline(file, line))
+        {
+            // Extract tag and content
+            if (std::regex_match(line, xmlTagMatch, xmlTagRegex))
+            {
+                const std::string& tag{ xmlTagMatch[1].str() };
+                std::string content{ xmlTagMatch[2].str() };
+
+                const long unsigned int found = content.find('\r');
```

Comment:
the correct type for a `find` function is `size_t` not ` const long unsigned int`